site/zenodo_rdm/files.py

```diff
@@ -149,3 +151,21 @@ def send_file(
 def storage_factory(**kwargs):
     """Create custom storage factory to enable file offloading."""
     return pyfs_storage_factory(filestorage_class=EOSFilesOffload, **kwargs)
+
+
+def checksum_verification_files_query():
+    """Return a FileInstance query taking into account file URI prefixes."""
+    files = FileInstance.query
+
+    uri_prefixes = current_app.config.get(
+        "FILES_REST_CHECKSUM_VERIFICATION_URI_PREFIXES"
+    )
+    if uri_prefixes:
+        files = files.filter(
+            sa.or_(*[FileInstance.uri.startswith(p) for p in uri_prefixes])
+        )
+    # Filtering out .ptif media files
+    excluded_path = current_app.config["IIIF_TILES_STORAGE_BASE_PATH"]
+    files = files.filter(~FileInstance.uri.contains(excluded_path))
```

Comment:
```suggestion
    files = files.filter(~FileInstance.uri.startswith(excluded_path))
```

minor: I think this will make for a more efficient SQL query also, i.e. the difference between `LIKE '%/images/iiif/%` vs `LIKE `/images/iiif/%`.